src/form/Select/MultiSelect.story.tsx

```diff
@@ -4,6 +4,8 @@ import { SelectOption } from './SelectOption';
 import { SelectMenu } from './SelectMenu';
 import { SelectInput, SelectInputChip } from './SelectInput';
 
+import CheckIcon from '@/assets/icons/check_circle.svg?react';
```

Comment:
Are we using the react svgr anywhere else? I was trying to avoid it at first and just inlining the SVGs not sure where we ended up tho

Reply:
<img width="800" alt="Screenshot 2024-06-11 at 12 41 34" src="https://github.com/reaviz/reablocks/assets/1290843/a548de13-7a2b-4c8d-9829-34470d8d4a28">